crates/algorithms/src/lib.rs

```diff
@@ -41,8 +42,9 @@ pub mod point_in_convex_hull;
 /// A K-Dimensional Tree data structure, useful for various geo-spatial computations.
 pub mod kd_tree;
 
+///impl Haversine formula to calc distance between to points and function to calculate bearing between to points
```

Comment:
/// Implementation of the Haversine formula, which calculate the distance and bearing between two points